package.json

```diff
@@ -88,7 +93,14 @@
     "stylelint-scss": "3.5.4",
     "stylelint-webpack-plugin": "0.10.5",
     "supertest": "3.4.2",
+    "ts-loader": "5.4.5",
+    "tslint": "5.16.0",
```

Comment:
We don't need `tslint` for linting. All project (since last week) are already using the centralized [eslint-config-verdaccio](https://github.com/verdaccio/eslint-config-verdaccio) `eslint` configuration compatible with Typescript. 

You can see here an example.

https://github.com/verdaccio/verdaccio-htpasswd/blob/master/.eslintrc

This will simplify all dependencies across all projects.